man/systemd.timer.xml

```diff
@@ -170,6 +170,12 @@
         expressions. Otherwise, the semantics are similar to
         <varname>OnActiveSec=</varname> and related settings.</para>
 
+        <para>Note that your service will only be activated by the timer
+        if the service is inactive. This means that for a
+        <varname>oneshot</varname> if you set <varname>RemainAfterExit=</varname>
```

Comment:
It makes less sense for a service that is meant to keep running since the service will stay running after it's been started, and thus won't be reactivated by the timer.